### PR TITLE
slight refactor to use of FriendlyName function

### PR DESCRIPTION
**Pull request by ArielLima**
### Which issue this PR addresses:

slight refactor to use of FriendlyName function

### CODE REVIEW

File: pkg/monitor/cluster/cluster.go [hunk 1]
```diff
@@ -120,8 +118,8 @@ func (mon *Monitor) Monitor(ctx context.Context) (errs []error) {
 	statusCode, err := mon.emitAPIServerHealthzCode(ctx)
 	if err != nil {
 		errs = append(errs, err)
-		mon.log.Printf("%s: %s", runtime.FuncForPC(reflect.ValueOf(mon.emitAPIServerHealthzCode).Pointer()).Name(), err)
-		mon.emitGauge("monitor.clustererrors", 1, map[string]string{"monitor": runtime.FuncForPC(reflect.ValueOf(mon.emitAPIServerHealthzCode).Pointer()).Name()})
+		mon.log.Printf("%s: %s", steps.FriendlyName(mon.emitAPIServerHealthzCode), err)
+		mon.emitGauge("monitor.clustererrors", 1, map[string]string{"monitor": steps.FriendlyName(mon.emitAPIServerHealthzCode)})
```
Comment (bennerv) [inline pkg/monitor/cluster/cluster.go:122]: nit:
generate the value once, use it in two different places.  
```suggestion
                friendlyName := steps.FriendlyName(mon.emitAPIServerHealthzCode)
		mon.log.Printf("%s: %s", friendlyName, err)
		mon.emitGauge("monitor.clustererrors", 1, map[string]string{"monitor": friendlyName})

**Review by bennerv — DISMISSED** [review]
lgtm - only one nit

@AldoFusterTurpin curious on your thoughts here as well.

File: pkg/monitor/cluster/cluster.go (hunk 1 above)
Comment (AldoFusterTurpin) [reply pkg/monitor/cluster/cluster.go:122]: Agree 100%. It is the same that I did in this [PR](https://github.com/Azure/ARO-RP/pull/2211/files) that you can see [here](https://github.com/Azure/ARO-RP/blob/master/pkg/monitor/cluster/cluster.go#L152) in this same file.

**Review by AldoFusterTurpin — DISMISSED** [review]
LGTM. Thanks!

**Review by bennerv — APPROVED** [review]
lgtm

**Pull request merged**